integration_test/install_test.go

```diff
@@ -25,3 +34,42 @@ func TestKrewInstall(t *testing.T) {
 	test.WithIndex().Krew("install", validPlugin).RunOrFailOutput()
 	test.AssertExecutableInPATH("kubectl-" + validPlugin)
 }
+
+func TestKrewInstall_Manifest(t *testing.T) {
```

Comment:
I think the other PR got mixed in here :)

Reply:
gosh, how embarrassing :roll_eyes: